src/backend/marsha/core/views.py

```diff
@@ -815,25 +815,36 @@ def get(self, request, *args, **kwargs):
             generated from applying the data to the template
 
         """
-        icon_url = ""
-        if settings.LTI_CONFIG_ICON:
+        model = kwargs.get("model")
+        launch_url = request.get_host()
+        title = settings.LTI_CONFIG_TITLE
+        icon_url = settings.LTI_CONFIG_ICON
+        if model in ["videos", "classrooms"]:
```

Comment:
if not you should raise a `404` exception

Reply:
If not we are sending regular config file.